internal/permissions/permission_store_test.go

```diff
@@ -14,109 +13,123 @@ func TestBasicPolicy1(t *testing.T) {
 	require.NoError(t, err)
 
 	// glob match on record key
-	ok, err := ps.HasPermission("did:1", "app.bsky.likes", "like1")
+	ok, err := ps.HasPermission("did:1", "did:owner", "app.bsky.likes", "like1")
```

Comment:
can we simplify this example and just have `did:1` and `did:2` or `did:owner` and `did:grantee` or something? it's a little confusing rn and i think we lost some test cases